File: EJ7/Persona.php
<?php
    class Persona {
        public $nombre;
        public $edad;
        public $genero;

        public function presentar() {
            echo "Hola, soy " .$this->nombre. ", tengo " .$this->edad." años y soy un/a " .$this->genero. "\n";
        }
    }

    $persona = new Persona();
    $persona->nombre = "Raul";
    $persona->edad = "18";
    $persona->genero = "Helicóptero Soviético";

    $persona->presentar();
?>

File: Ejercicio4/EJ1/misPaquetes/Persona.java
package misPaquetes;

public class Persona {
	String nombre;
	int edad;
	
	public Persona(String nombre, int edad) {
		this.nombre = nombre;
		this.edad = edad;
	}
	
	void mostrarDatos() {
		System.out.println("Hola, mi nombre es " + nombre + " y tengo " + edad + " años");
	}
}
